src/components/ExploreList.jsx

```diff
@@ -1,29 +1,28 @@
-import React, {useState, useEffect} from 'react'
+import React, { useState, useEffect } from 'react'
 import { Item, Segment, Header, Message } from 'semantic-ui-react'
 import {getExploreArticles} from '../modules/articlesDataModule'
 
-const ExploreList = () => {
+const ExploreList = ({articleType}) => {
```

Comment:
Refactor this to use Redux, so you don't have to pass props but mainly for experience and get going with it